locale/id/LC_MESSAGES/django.po

```diff
@@ -229,21 +240,25 @@ msgstr ""
 
 #: gem/templates/profiles/register.html:15
 msgid "Choose a username"
-msgstr "Pilih suatu nama pengguna:"
+msgstr "Pilih suatu nama pengguna"
 
 #: gem/templates/profiles/register.html:19
 msgid "Choose a 4-digit pin"
 msgstr "Pilih 4 digit pin"
 
 #: gem/templates/profiles/register.html:23
-msgid "I identify my gender as (only you will see this)"
-msgstr "Saya berjenis kelamin (hanya saya yang dapat melihat hal ini)"
+msgid "I identify my gender as"
+msgstr "Saya berjenis kelamin"
+
+#: gem/templates/profiles/register.html:26
+msgid "Only you will see this"
+msgstr "hanya saya yang dapat melihat hal ini"
```

Comment:
shouldn't this be capitalized? As far as I can see neither the templates nor the css capitalize it.